com.pojtinger.felicitas.Multiplex.json

```diff
@@ -0,0 +1,648 @@
+{
+  "app-id": "com.pojtinger.felicitas.Multiplex",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.golang"],
+  "command": "multiplex",
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=xdg-download:rw",
+    "--filesystem=xdg-videos:rw",
+    "--filesystem=/tmp:rw",
+    "--talk-name=org.freedesktop.Flatpak"
+  ],
+  "modules": [
+    {
+      "name": "app",
```

Comment:
a more descriptive name is nicer.

Reply:
Switched to "gui" and "dependencies" for the two modules in 94a3be1f512d76bcd40deb8708fe3c38658a4bcc; that should make them a bit clearer hopefully.

Reply:
really? It's not better at all. A proper name would simply be `multiplex` since that the name of the app.

Reply:
Why would that be necessary? The module name is relative to the manifest's ID as in `com.pojtinger.felicitas.Multiplex/gui`, `com.pojtinger.felicitas.Multiplex/dependencies`. Doing `com.pojtinger.felicitas.Multiplex/multiplex` feels like its an unnecessary duplication (being that we can use `$FLATPAK_ID` everywhere but here so as to not have to copy-and-paste it around), and it also makes adapting it for another up unnecessarily hard.

If its a convention to do that duplication though then sure I can change it since this is probably close to bikeshedding :)

Reply:
> Why would that be necessary? 

Becaus when maintainer go awol it's people like myself that end up picking up the pieces. So better make this less unpleasant by having things nicely done from the get go. Call that bikeshedding like you want, I build car garages.

Reply:
Sure, fixed in ee7a56fd23bf685107ae0fe6b048ba682190dc63